ui/raidboss/data/05-shb/trial/varis-ex.txt

```diff
@@ -0,0 +1,116 @@
+45.1 "Citius" sync /:Varis Yae Galvus:4CF0:/
+56.0 "Ignis Est Ready" sync /:Varis Yae Galvus:4CB5:/
+64.2 "Ignis Est Action"
+64.2 "Festina Lente" sync /:Varis Yae Galvus:4CC9:/
+82.4 "Electrified Gunshield" sync /:Varis Yae Galvus:4CD7:/
+91.5 "Altius" sync /:Varis Yae Galvus:4CCA:/
+100.4 "Ignis Est Ready" sync /:Varis Yae Galvus:4CB5:/
+101.6 "Terminus Est" # sync /:Terminus Est:4CB4:/
+108.5 "Ignis Est Action" # sync /:Ignis Est:4CB6:/
+109.6 "Magitek Shock" sync /:Varis Yae Galvus:4CDA:/
+120.7 "Reinforced Gunshield" sync /:Varis Yae Galvus:4CD9:/
+128.9 "Citius" sync /:Varis Yae Galvus:4CF0:/
+132.0 "Magitek Shielding" sync /:Varis Yae Galvus:4CDB:/
+147.2 "Loaded Gunshield" sync /:Varis Yae Galvus:4CD8:/
+156.0 "Ventus Est Ready" sync /:Varis Yae Galvus:4CC6:/
+163.6 "Magitek Burst" sync /:Varis Yae Galvus:4CDE:/
+164.2 "Ventus Est Action"
+166.7 "Magitek Burst Hit"
+177.1 "Vivere Militare Est" sync /:Varis Yae Galvus:4CCC:/
+
+# Phase 2
+200.1 "--targetable--"
+233.2 "Vivere Militare Est" sync /:Bladesblood:4CCF:/ duration 30 window 80,50
+
+# Phase 3 (random groups)
+262.3 "Gunshield Actions" # fake action here for placeholding
+
+
+# ================= random groups ===================
+# group 1 (Loaded Gunshield, spread)
+300.0 "Loaded Gunshield" sync /:Varis Yae Galvus:4CD8:/ window 100,400
+308.9 "Ignis Est Ready" sync /:Varis Yae Galvus:4CB5:/
+313.2 "Magitek Burst" sync /:Varis Yae Galvus:4CDE:/
+314.7 "Terminus Est"
+316.3 "Magitek Burst" sync /:Varis Yae Galvus:4CDF:/
+317.0 "Ignis Est Action"
+324.3 "Terminus Est" # sync /:Terminus Est:4CB4:/
+324.5 "Festina Lente" sync /:Varis Yae Galvus:4CC9:/
+# since trigger would notice tankbuster with startsUsing, timeline would not show it
+# 312.7 "Citius" sync /:Varis Yae Galvus:4CF0:/
+
+
+# group 2 (Reinforced Gunshield, block)
+500.0 "Reinforced Gunshield" sync /:Varis Yae Galvus:4CD9:/ window 300,300
+509.1 "Altius" sync /:Varis Yae Galvus:4CCA:/
+518.0 "Ventus Est Ready" sync /:Varis Yae Galvus:4CC6:/
+519.2 "Terminus Est" # sync /:Terminus Est:4CB4:/
+521.3 "Magitek Shielding" sync /:Varis Yae Galvus:4CDB:/
+526.1 "Ventus Est Action" sync /:Ventus Est:4CC7:/
+
+# group 3 (Electrified Gunshield, knockback)
+700.0 "Electrified Gunshield" sync /:Varis Yae Galvus:4CD7:/ window 500,100
+705.1 "Reinforcements" sync /:Varis Yae Galvus:4CEA:/
+711.0 "Terminus Est" # sync /:Phantom Varis:4CB3:/
+714.2 "Aetherochemical Grenado" sync /:Magitek Turret II:4CED:/
+723.9 "Terminus Est" # sync /:Terminus Est:4CB4:/
+726.6 "Magitek Shock" sync /:Varis Yae Galvus:4CDA:/
+737.5 "Alea Iacta Est" sync /:Varis Yae Galvus:4CD2:/
+# ==================== random groups ==================
+
+
+# Phase 4 
+1000.0 "--sync--" sync /:Varis Yae Galvus:4CE0:/ window 800,20
+1010.4 "Magitek Spark/Torch" sync /:Gunshield:(4CE4|4CE3):/ window 10,5
+1019.1 "Magitek Torch/Spark" sync /:Gunshield:(4CE4|4CE3):/ window 5,5
+
+# Phase 5
+1056.8 "Loaded Gunshield" sync /:Varis Yae Galvus:4CD8:/
```

Comment:
Oh, running your timeline through `test_timeline`, I'm noticing that this doesn't sync, and the rest of the timeline seems to be lost. Could you add a `window 56.8,5` to ensure the timeline continues from here?

Reply:
OK, I forgot that there should be a `window` when the phase changing. ( seems like the DPS of my party was a little low so that I have not notice that issue.)
I have pushed it.